### PR TITLE
Fix: Customizer error

### PR DESCRIPTION
**Pull request by Balachandark**

### CODE REVIEW

File: inc/customizer/custom-controls/background/background.js [hunk 1]
```diff
@@ -1,191 +1,193 @@
...
+				// Hide unnecessary controls.
+				control.container.find( '.background-wrapper > .background-repeat' ).toggle();
+				control.container.find( '.background-wrapper > .background-position' ).toggle();
+				control.container.find( '.background-wrapper > .background-size' ).toggle();
+				control.container.find( '.background-wrapper > .background-attachment' ).toggle();
+
+				if( 'down' === $(this).attr( 'data-direction' ) )
+				{
+					$(this).attr('data-direction', 'up');
+					$(this).find('.message').html( astraCustomizerControlBackground.lessSettings )
```
Comment (UttamSharma007) [inline inc/customizer/custom-controls/background/background.js]: Semicolor?

File: inc/customizer/custom-controls/background/background.js [hunk 2]
```diff
@@ -1,191 +1,193 @@
...
+					$(this).find('.icon').html( '↑' );
+				} else {
+					$(this).attr('data-direction', 'down');
+					$(this).find('.message').html( astraCustomizerControlBackground.moreSettings )
```
Comment (UttamSharma007) [inline inc/customizer/custom-controls/background/background.js]: Semicolor?

File: inc/customizer/custom-controls/background/background.js (hunk 1 above)
Comment (Balachandark) [reply inc/customizer/custom-controls/background/background.js]: Added semicolons

File: inc/customizer/custom-controls/background/background.js (hunk 2 above)
Comment (Balachandark) [reply inc/customizer/custom-controls/background/background.js]: Added semicolons

**Pull request merged**